staging/src/k8s.io/apiserver/pkg/endpoints/responsewriter/testing/fake.go

```diff
@@ -89,3 +89,26 @@ func AssertResponseWriterInterfaceCompatibility(t *testing.T, inner, outer http.
 		t.Errorf("Expected the inner and outer http.ResponseWriter object to be compatible with http.Hijacker, but got - inner: %t, outer: %t", innerHijackable, outerHijackable)
 	}
 }
+
+func AssertResponseWriter(t *testing.T, w http.ResponseWriter, req *http.Request) {
```

Comment:
assert what?

Reply:
renamed to `AssertResponseWriterImplementsExtendedInterfaces`